src/wal/test/waltest.c

```diff
@@ -36,7 +37,7 @@ int writeToQueue(void *pVnode, void *data, int type, void *pMsg) {
 }
 
 int main(int argc, char *argv[]) {
-  char path[128] = "/home/jhtao/test/wal";
+  char path[128] = {0};
```

Comment:
We need a non null default value here for help (./waltest -h) printing.
Any other fixes look good to me.

Reply:
sure, will fix.

Reply:
I put "/tmp/wal" as default value in the second iteration.